lib/tizen_sdk.dart

```diff
@@ -250,22 +250,36 @@ class TizenSdk {
     String? apiVersion,
     required String arch,
   }) {
-    if (profile == 'common') {
-      // Note: The headless profile is not supported.
-      profile = 'iot-headed';
-    }
-    if (profile == 'tv') {
-      // Note: The tv-samsung rootstrap is not publicly available.
-      profile = 'tv-samsung';
-    }
     apiVersion ??= '6.0';
+    if (apiVersion == '5.5') {
+      throwToolExit('The $apiVersion version is not supported.');
+    }
 
-    double versionToDouble(String versionString) {
-      final double? version = double.tryParse(versionString);
-      if (version == null) {
-        throwToolExit('The API version $versionString is invalid.');
+    if (profile == 'tizen') {
+      if (apiVersion != '8.0') {
+        throwToolExit(
+            'The $apiVersion version is not supported by the tizen profile.');
+      }
+    } else if (profile == 'common') {
+      if (apiVersion == '8.0') {
+        // Note: Starting with Tizen 8.0, the unified "tizen" profile is used.
+        profile = 'tizen';
+      } else {
+        // Note: The headless profile is not supported.
+        profile = 'iot-headed';
+      }
+    } else if (profile == 'tv' || profile == 'tv-samsung') {
+      // Note: The tv-samsung and the tv rootstrap is not publicly available.
+      if (apiVersion == '8.0') {
+        profile = 'tizen';
+      } else {
+        profile = 'iot-headed';
```

Comment:
Why... why?? The common rootstrap should be used as a fallback if and only if the tv-samsung rootstrap is not found.

Reply:
um.. I installed almost all the profiles but couldn't find any case where 'tv' has rootstrap.
So I thought the case of finding TV rootstrap was unnecessary.
How do I test a case using 'tv' rootstrap?
```
~/tizen-studio/platforms 19:41:45 $ find -name "*" -exec sh -c 'find "{}" -name "rootstraps" -print' \;
./tizen-7.5/mobile/rootstraps
./tizen-7.5/tizen-headed/rootstraps
./tizen-6.0/iot-headed/rootstraps
./tizen-6.0/iot-headless/rootstraps
./tizen-6.0/mobile/rootstraps
./tizen-6.0/wearable/rootstraps
./tizen-6.5/iot-headed/rootstraps
./tizen-6.5/iot-headless/rootstraps
./tizen-6.5/mobile/rootstraps
./tizen-6.5/wearable/rootstraps
./tizen-7.0/iot-headed/rootstraps
./tizen-7.0/iot-headless/rootstraps
./tizen-7.0/mobile/rootstraps
./tizen-7.0/wearable/rootstraps
./tizen-8.0/tizen/rootstraps
...
...
```

Reply:
The `tv-samsung` rootstrap is not publicly available. You can get one (`rootstraps.zip`) from the VD image repository (inhouse).